examples/bq-email-exports/deploy.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SA_NAME="email-exports-service-account"
+SVC_DESC="The service account used by the BQ email export Cloud Function"
+PROJECT_ID="report-scheduling"
+
+# Create Cloud Functions service account
+gcloud iam service-accounts create $SA_NAME --description "$SVC_DESC" --project "$PROJECT_ID"
+SVC_ACCOUNT_EMAIL=$(gcloud iam service-accounts list --filter="name:$SA_NAME" --format "value(email)")
+
+CF_NAME="email-export-function"
+CS_NAME="bq-email-scheduler"
+TOPIC_NAME="bq-exports"
+TOPIC_PATH="projects/${PROJECT_ID}/topics/bq-exports"
+SENDGRID_API_KEY="{$API_KEY}"
+SCHEDULE="00 00 * * *"
+SCHEDULER_MESSAGE="BigQuery email export"
```

Comment:
We can do a lot more with this!
Make this a useful JSON string that configures everything (see comments below).
The reasoning for this is you can deploy 1 reusable cloud function (which is like a micro-service for scheduling query result emails) and then have many cloud scheduler jobs with different payloads to schedule different queries.

Reply:
Agreed! Did this in the new commit.